mock/mock-functions.js

```diff
@@ -1,3 +1,13 @@
+const onFdc3Ready = () => {
```

Comment:
Being very pedantic ... you don't need the extra braces and `return` statement:

~~~
const onFdc3Ready = () => new Promise(...)
~~~